exercises/transpose/src/test/java/TransposeTest.java

```diff
@@ -0,0 +1,259 @@
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.Before;
+
+import static org.junit.Assert.assertEquals;
+
+public class TransposeTest {
+    private Transpose transpose;
+
+    @Before
+    public void setup() {
+        transpose = new Transpose();
+    }
+
+    @Test
+    public void emptyString() {
+        String input = "";
+        String expected = "";
+
+        assertEquals(expected, transpose.transpose(input));
+    }
+
+    @Test
+    @Ignore("Remove to run test")
+    public void twoCharsInARow() {
+        String input = "A1";
+        String expected = "A" +
+                "\n1";
```

Comment:
The indentation here makes this a bit difficult to read. Could you maybe try and align the rows of the matrices so it looks more like a matrix?